Add "Adding plugins" section (#123)
Quaver/Quaver.Wiki: en-US/editor/plugins.md

        * plugin.lua
        * settings.ini

Each individual folder that contains a `plugin.lua` and a `settings.ini` file
represents a separate plugin.

*Adding* a plugin is done by adding a folder that contains a `plugin.lua` file
and a `settings.ini` file to your `Quaver/Plugins` folder. It should show up in
your Plugins dropdown next time you open the editor. Do keep in mind that the
plugin will show up with the name specified in the `settings.ini` file, and not

> the name of the folder.

Reviewer: ```suggestion
*Adding* a plugin is done by adding a folder that contains a `plugin.lua` file
and a `settings.ini` file to your `Quaver/Plugins` folder. It should show up in
your Plugins dropdown the next time you open the editor. Do keep in mind that the
plugin will show up with the name specified in the `settings.ini` file and not
the name of the folder.
```

> However, the process of *getting* a plugin can be slightly different, depending
> on where the plugin is hosted or shared. For example, in case the plugin is
> hosted on GitHub, you want to click on the green "Download Code" button and
> download as a ZIP file, then unzip the file to get a folder.

Reviewer: ```suggestion
However, the process of *getting* a plugin can be slightly different depending
on where the plugin is hosted or shared. In the case that the plugin is
hosted on GitHub, you can click on the green "Download Code" button, and
download it as a ZIP file. Then unzip the file to get a folder.
```
